src/shared/styles/components/add-contact-modal.css

```
@@ -44,63 +46,47 @@
 .add-contact-modal .direct-actions {
   display: flex;
-  flex-direction: row;
-  justify-self: center;
-  gap: 0.75rem;
-  margin-bottom: 0;
+  flex-direction: column;
+  justify-content: space-around;
+  gap: 1.5rem;
 
-  width: clamp(220px, 95%, 600px);
+  margin: 0 auto;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)